Lib/test/test_asyncio/test_locks.py

```diff
@@ -758,6 +758,63 @@ async def test_timeout_in_block(self):
             with self.assertRaises(asyncio.TimeoutError):
                 await asyncio.wait_for(condition.wait(), timeout=0.5)
 
+    async def test_cancelled_error_wakeup(self):
+        """Test that a cancelled error, received when awaiting wakeup
+        will be re-raised un-modified.
+        """
```

Comment:
Please convert the docstrings on the tests to comments -- test docstrings tend to be printed by the test runner, messing up the neat output. Note how no other tests here have docstrings.